src/CommentSold/Api/Resources/Request/Product/CreateProductRequest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CommentSold\Api\Resources\Request\Product;
+
+use CommentSold\Api\Resources\Request\AbstractRequest;
+
+class CreateProductRequest extends AbstractRequest
+{
+    public string $name;
+    public ?string $description = null;
+    public string $style;
+    public ?string $brand = null;
+    public bool $charge_taxes;
+    public ?string $tax_code = null;
+    public ?int $shipping_price_in_cents = null;
+    public bool $is_archived;
+    public ?int $quantity_per_user_limit = null;
+    public ?int $quantity_per_stream_per_user_limit = null;
+    public ?Attributes $attribute_names = null;
+    public ?Image $main_image = null;
+    /** @var array[Image]|null */
+    public ?array $additional_images = null;
+    public ?bool $rehost_images = null;
+    public ?int $shopify_id = null;
+    /** @var array[string]|null */
+    public ?array $tags = null;
+    public ?int $category = null;
+    /** @var array[Variant] */
+    public array $variants;
+
+    public function __construct(array $payload)
+    {
+        $additional_images = [];
+        if (isset($payload['additional_images'])) {
+            foreach ($payload['additional_images'] as $image) {
+                $additional_images[] = new Image($image);
+            }
+        }
```

Comment:
```suggestion
        foreach ($payload['additional_images'] ?? [] as $image) {
            $additional_images[] = new Image($image);
        }
```
just some syntactic sugar